### DISCUSSION TITLE
Haystack DocumentStore Integration

### DISCUSSION THREAD

**Discussion by jkfnc** [meilisearch/product#732] [Feedback & Feature Proposal]
Would be good to have Haystack DocumentStore, as it seems to be a good alternative to langchain.

**Comment by macraig** [comment]
Hi @jkfnc , thank you for submitting your proposal. Are you currently using Haystack? I'd love to know more about how it fits your use case and why you chose it over Langchain.

**Reply by jkfnc** [reply]
@macraig Just experimenting with haystack for now, feel like the abstractions are better , less confusing and clean compared to langchain in haystack 2.0 .
